docs/sources/reference/components/otelcol/otelcol.processor.groupbyattrs.md

```diff
@@ -0,0 +1,88 @@
+# otelcol.processor.groupbyattrs
+
+`otelcol.processor.groupbyattrs` accepts telemetry data from other `otelcol`
+components and reassociates spans, log records and metric datapoints to a Resource that matches with the specified attributes. It groups telemetry data by specified attributes.
+
+{{% admonition type="note" %}}
+`otelcol.processor.groupbyattrs` is a wrapper over the upstream OpenTelemetry
+Collector `groupbyattrs` processor.  If necessary, bug reports or feature requests
+will be redirected to the upstream repository.
+{{% /admonition %}}
+
+It is recommended to use the groupbyattrs processor together with batch processor, as a consecutive step, as this will reduce the fragmentation of data (by grouping records together under matching Resource/Instrumentation Library)
+
+You can specify multiple `otelcol.processor.groupbyattrs` components by giving them
+different labels.
+
+## Usage
+
+```river
+otelcol.processor.groupbyattrs "LABEL" {
+  output {
+    metrics = [...]
+    logs    = [...]
+    traces  = [...]
+  }
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+Name | Type            | Description                                                                           | Default    | Required
+---- |-----------------|---------------------------------------------------------------------------------------|------------| --------
+`keys` | `array(string)` | Keys that will be used to group the spans, log records or metric data points together | | no
+
+
+## Exported fields
+
+The following fields are exported and can be referenced by other components:
+
+Name    | Type               | Description
+--------|--------------------|-----------------------------------------------------------------
+`input` | `otelcol.Consumer` | A value that other components can use to send telemetry data to.
```

Comment:
```suggestion
`input` | `otelcol.Consumer` |  Accepts `otelcol.Consumer` data for metrics, logs, or traces.
```

I'm not sure about this description. I don't think components can send data to a value... what if we pulled up the extended description. If this works, we could remove lines 46 and 47.

Reply:
I'll look into that

Reply:
There is no extended description for this section. All the files have copies of the same thing. It is a good ideaa to make this part into a separate file. I would assume it is the same for all otelcol processors